scripts/generate_climos

```diff
@@ -89,7 +101,11 @@ if __name__ == "__main__":
         choices=["mean", "std"],
     )
     parser.set_defaults(
-        dry_run=False, convert_longitudes=True, split_vars=True, split_intervals=True
+        dry_run=False,
+        dry_output_to_file=False,
+        convert_longitudes=True,
+        split_vars=True,
+        split_intervals=True,
     )
```

Comment:
I know this was here before, but I would suggest moving these defaults into their definitions. I'll leave this one up to you though, whatever you feel is more clear.